src/app/backend/namespacelist.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// List of namespaces in the cluster.
+type NamespaceList struct {
+	// Unordered list of namespaces.
+	Namespaces []Namespace `json:"namespaces"`
+}
+
+// Kubernetes namespace representation.
+type Namespace struct {
```

Comment:
I think that a separate struct is not needed for namespaces. Even in kubernetes API is is just a string. So how about removing this struct and making `Namespaces` a slice of strings?